src/prebuild-index.html

```diff
@@ -14,5 +14,12 @@
 
 <body>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/polyglot.js/2.2.2/polyglot.min.js"></script>
+  <script>
+    window.fwSettings={
+    'widget_id':48000002116
+    };
+    !function(){if("function"!=typeof window.FreshworksWidget){var n=function(){n.q.push(arguments)};n.q=[],window.FreshworksWidget=n}}() 
```

Comment:
I don't really understand the minified JS here. Was that supplied by Freshworks?

Reply:
I think so @forevermatt. I got it from Kate aka @kode-monki so I've added her.

Reply:
yep, straight from Freshworks